guide/src/test/kotlin/examples/example-recursive-03.kt

```diff
@@ -3,22 +3,20 @@ package arrow.website.examples.exampleRecursive03
 
 import io.kotest.matchers.shouldBe
 
-import arrow.core.memoize
-
-val fibonacciMemoized = ::fibonacciWorker.memoize()
+import arrow.core.MemoizedDeepRecursiveFunction
 
+val fibonacciWorker = MemoizedDeepRecursiveFunction<Int, Int> { n ->
```

Comment:
It's not a big deal but wouldn't it be more appropriate to name this function "fibonacciMemoized" instead of "fibonacciWorker"?

Reply:
That actually makes sense